Misc/NEWS.d/next/Core and Builtins/2021-08-31-11-09-52.bpo-45012.ueeOcx.rst

```diff
@@ -0,0 +1 @@
+DirEntry object returned by os.scandir release GIL around stat, lstat and fstatat syscalls.
```

Comment:
One more thing: please use reST markup in NEWS entries.

```suggestion
In :mod:`posixmodule`, release GIL during ``stat()``, ``lstat()``, and
``fstatat()`` syscalls. Patch by Stanisław Skonieczny.
```

See the [devguide](https://devguide.python.org/documenting/#restructuredtext-primer) for a reST primer :)